src/patternfly/components/Tabs/tabs.scss

```diff
@@ -231,6 +232,11 @@ $pf-c-tabs--spacer-map: build-spacer-map("none", "sm", "md", "lg", "xl", "2xl");
       --pf-c-tabs__link--before--BorderBottomWidth: var(--pf-c-tabs__link--before--BorderWidth);
     }
 
```

Comment:
it's fine the way it is, but looks like you could use this here

```
.pf-c-tabs__item {
  &:first-child{}
  &:last-child {}
  &.pf-m-current{}
}
```

Reply:
👍 https://github.com/patternfly/patternfly/issues/3052